### PR TITLE
Add Ionia

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

 none

### Dependencies

 none

### Description

 none

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204732629745374

### CODE REVIEW

File: src/partners/ioniavisarewards.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+type IoniaStatus = ReturnType<typeof asIoniaStatus>
+
+const MAX_RETRIES = 5
+const LIMIT = 200
+const QUERY_LOOKBACK = 1000 * 60 * 60 * 24 * 5 // 5 days
+
+const statusMap: { [key in IoniaStatus]: Status } = {
+  complete: 'complete',
+  other: 'complete'
+}
```
Comment (swansontec) [inline src/partners/ioniavisarewards.ts:47]: Same thing for this copy-pasted status stuff, which isn't needed.

**Review by swansontec — APPROVED** [review]
Approved, since it certainly works, but the status code really needs to be deleted.

File: src/partners/ioniavisarewards.ts (hunk 1 above)
Comment (paullinator) [reply src/partners/ioniavisarewards.ts:47]: It’s really there as a template for when Ionia adds statuses. Their API is still being modified per our feedback and that’s one of the pending changes.

Comment (swansontec) [reply src/partners/ioniavisarewards.ts:47]: Ok, I didn't have that bit of context. My preference is to keep things clean until they are needed, and then copy / paste statuses from another plugin, but I see how this way makes sense too. Merge away.

**Pull request merged**